parser.py

```diff
@@ -0,0 +1,45 @@
+# Author: Pankil Shah
+# Created on: 03/13/2021
+# Description: Parse a corpus and dump the sentences into the database 
+
+from nltk.corpus import brown
+import nltk.data
+import logic
+import datetime
+from os import listdir
+from os.path import isfile, join
+
+#print(brown.tagged_sents()[0:2])
```

Comment:
Can we take out comments that do not contribute the understanding of the file?

Reply:
Sure, I'll do that.